misc/test/examples_test.go

```diff
@@ -148,6 +148,53 @@ func TestExamples(t *testing.T) {
 				})
 			},
 		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:       path.Join(cwd, "..", "..", "cloud-js-containers"),
+			SkipBuild: true,
+			Config: map[string]string{
+				// Use us-west-2 to assure fargate is availablecd
+				"aws:region":           "us-west-2",
+				"cloud-aws:useFargate": "true",
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				assertHTTPResult(t, stack.Outputs["hostname"], func(body string) bool {
+					return assert.Contains(t, body, "<p>Hello, containers!</p>")
+				})
+			},
+		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:       path.Join(cwd, "..", "..", "cloud-js-httpserver"),
+			SkipBuild: true,
+			Config: map[string]string{
+				"cloud:provider": "aws",
+				"aws:region":     awsRegion,
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				assertHTTPResult(t, stack.Outputs["endpoint"].(string)+"/hello", func(body string) bool {
+					return assert.Contains(t, body, "{\"route\":\"/hello\",\"count\":1}")
+				})
+			},
+		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:           path.Join(cwd, "..", "..", "cloud-ts-url-shortener-cache-http"),
+			Verbose:       true,
+			DebugLogLevel: 8,
+			DebugUpdates:  true,
+			SkipBuild:     true,
+			Config: map[string]string{
+				"url-shortener-cache:redisPassword": "s3cr7Password",
+				"cloud:provider":                    "aws",
+				// Use us-west-2 to assure fargate availability
+				"aws:region":                     "us-west-2",
+				"cloud-aws:useFargate":           "true",
+				"cloud-aws:functionIncludePaths": "",
```

Comment:
I am not sure, it is in the example readme (https://github.com/pulumi/examples/tree/master/cloud-ts-url-shortener-cache-http)

6.  Add the 'www' directory to the uploaded function code so it can be served from the http server:

$ pulumi config set cloud-aws:functionIncludePaths
#or
$ pulumi config set cloud-azure:functionIncludePaths

Reply:
Hmm - sounds like this might be needed but that the comment in the readme was missing some text. Cc @cyrusnajmabadi.